tests/testthat/test-setup.R

```diff
@@ -74,8 +74,7 @@ test_that("`metadata_create_template` is working with simulated user input", {
   expect_no_error(
```

Comment:
should be `test_no_error`?

Reply:
@dfalster The custom `test_expect_` functions in `test_functions.R` are only used in `dataset_test` where customisation of error messages is needed. The rest of the tests (`test-setup.R`, `test-xamples.R`, etc.) use testthat:: functions -- which is why I renamed all the custom test functions to `test_` instead of keeping the `expect_` syntax to avoid confusion.